### PR TITLE
Sound system/playlist creation

### LINKED ISSUE #115: Character Manager throws exception when starting scene from main menu
image

### LINKED ISSUE #193: Load Playlists from DB

### LINKED ISSUE #197: The game hangs for a second when exiting the Starting Area Music Trigger Box
Probably caused by Unity loading a huge audio file from the disk.
Maybe load in parallel with coroutines or threads?

### PR DESCRIPTION
**Pull request by Segelzwerg**
This allows to easily add playlist without database from the Unity Editor. It also reworks and simplifies the music manager and the sound areas due to easier handling of playlists.

closes #115 
closes #193
closes #197

### CODE REVIEW

**Review by Mimonsi — CHANGES_REQUESTED** [review]
The Audio Settings are broken? Changing the music slider does not work at all.

File: Assets/Scenes/TestScene.unity [hunk 1]
```diff
@@ -202,7 +202,12 @@ MonoBehaviour:
   displayText: Delicious Cube
   displaySubtext: '[E] to pick up'
   ignoreTagCheck: 0
+<<<<<<< HEAD
+  itemToPickup: {fileID: 58653346}
+  inventory: {fileID: 143817151}
+=======
   pickupItem: {fileID: 58653346}
+>>>>>>> origin/master
```
Comment (Mimonsi) [inline Assets/Scenes/TestScene.unity]: Merge conflict unsolved?

**Review by Mimonsi — CHANGES_REQUESTED** [review]

File: Assets/Scenes/TestScene.unity (hunk 1 above)
Comment (Segelzwerg) [reply Assets/Scenes/TestScene.unity]: Nothing at my end. So I don't know.

Comment (Segelzwerg) [reply Assets/Scenes/TestScene.unity]: I did this with the tool and Github is not showing conflicts and I don't  to fix i

Comment (Mimonsi) [reply Assets/Scenes/TestScene.unity]: grafik

I still cannot open the scene, there are conflicts

**Review by Mimonsi — CHANGES_REQUESTED** [review]
grafik
Still merge conflicts :/

File: Assets/Scenes/TestScene.unity (hunk 1 above)
Comment (Segelzwerg) [reply Assets/Scenes/TestScene.unity]: I did some more research and the UnityYAML Merger didn't do a thing. So can some one with a working version do it for me?

**Comment by Segelzwerg** [comment]
thx @iTitus  for merging

**Review by iTitus — DISMISSED** [review]

File: Assets/Scripts/Areas/SoundArea.cs [hunk 2]
```diff
@@ -1,9 +1,39 @@
+using Sounds;
+using Sounds.Manager;
 using UnityEngine;
 
 namespace Areas
 {
-    public abstract class SoundArea : MonoBehaviour
+    public class SoundArea : MonoBehaviour
```
Comment (Mimonsi) [inline Assets/Scripts/Areas/SoundArea.cs:8]:
```suggestion
    [AddComponentMenu("Sound/SoundArea")
    public class SoundArea : MonoBehaviour
```

**Review by Mimonsi — CHANGES_REQUESTED** [review]
Looks very good. Could you Add `[AddComponentMenu("Sound/[ClassName")]` to the remaining MonoBehavior scripts? 

I would have done it, but my auto formatting makes everything really chaotic

File: Assets/Scripts/Areas/SoundArea.cs (hunk 2 above)
Comment (Segelzwerg) [reply Assets/Scripts/Areas/SoundArea.cs:8]: Sounds very good

**Review by Mimonsi — APPROVED** [review]

**Pull request merged**